tests/bindings/python/test_model.py

```diff
@@ -38,8 +38,163 @@ def test_model(self):
         del(x, y)
         
         # Test own methods
-        #TODO Models, components etc.
         
+        # void addUnits(const UnitsPtr &units)
+        from libcellml.units import Units
+        m = Model()
+        u = Units()
+        m.addUnits(u)
+        del(m, u)
+        
+        # bool removeUnits(size_t index)
+        m = Model()
```

Comment:
Why not break it down into different `test_model_feature` so that each test can be better documented and we don't need to worry about deleting variables.  This also allow tests to be independent of each other so that specific failure cases can be more easily identified when they happen.

Reply:
Ok!

Reply:
Done!